src/main/java/io/swagslash/gametrackerserver/endpoint/webapp/controller/WebappController.java

```diff
@@ -1,15 +1,41 @@
 package io.swagslash.gametrackerserver.endpoint.webapp.controller;
 
+import io.swagslash.gametrackerserver.dto.webapp.GameDTO;
+import io.swagslash.gametrackerserver.service.GameService;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/api/webapp")
 public class WebappController {
 
-    @RequestMapping({"/", ""})
-    public String index() {
-        return "test";
+    private GameService gameService;
+
+    public WebappController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @GetMapping("/games")
+    @PreAuthorize("hasRole('USER')")
+    public List<GameDTO> findAll() {
```

Comment:
IMHO, you should not be able to query all games regardless of the user.

Reply:
Method only exists due to testing will be removed